Adding Component#onMount (#228)
flame-engine/flame: doc/components.md

The `isHUD` method can be implemented to return true (default false) to make the `BaseGame` ignore the `camera` for this element.

> The `onMount` method can be overridden to run initializations code for the component, when this method is called, BaseGame has make sure that all the mixin which would change this component behaviour, is already resolved.

Reviewer: ```suggestion
The `onMount` method can be overridden to run initialization code for the component. When this method is called, BaseGame ensures that all the mixins which would change this component behaviour are already resolved.
```
